config/configs/repl_config.rb

```diff
@@ -0,0 +1,3 @@
+class ReplConfig < ApplicationConfig
+  attr_config url: "http://localhost:3001/", api_key: ""
+end
```

Comment:
There is [EditorConfig](https://github.com/rubyapi/rubyapi/blob/1c207e0/.editorconfig#L9). :cry: